src/ZkFold/Base/Algebra/Basic/Class.hs

```diff
@@ -123,6 +124,7 @@ multiExp a = foldl (\x y -> x * (a ^ y)) one
 
 ------------------------------- Roots of unity ---------------------------------
 
+    {--
```

Comment:
I would get rid of this code altogether as the implementation is already moved to `instance Prime p => Field (Zp p)`

Reply:
yeah, I just forgot to remove it.